crates/driver/src/infra/config/file/load.rs

```diff
@@ -44,7 +55,129 @@ pub async fn load(path: &Path) -> infra::Config {
             }
         }))
         .await,
-        liquidity: config.liquidity,
+        liquidity: liquidity::Config {
```

Comment:
this code just moved here from `run.rs` with `s/eth.network_id()/&network.id/g`.